array-manipulation-1/src/initial.js

```diff
@@ -0,0 +1,21 @@
+/* exported initial */
+
+function initial(array) {
+  var reversed = [];
+  for (var i = array.length - 1; i >= 0; i--) {
+    reversed.push(array[i]);
+  }
+  var newReveresed = [];
+  for (var y = 1; y < reversed.length; y++) {
+    newReveresed.push(reversed[y]);
+  }
+  var newArray = [];
+  for (var x = newReveresed.length - 1; x >= 0; x--) {
+    newArray.push(newReveresed[x]);
+  }
+  return newArray;
+}
```

Comment:
You don't need to use 3 different loops here. You only need 1 loop. You would just only have the loop run for all indexes except for the last. Here is our solution

```
function initial(array) {
  var output = [];
  for (var i = 0; i < array.length - 1; i++) {
    var element = array[i];
    output.push(element);
  }
  return output;
}
```

Reply:
bruh